modAionImpl/test/org/aion/zero/impl/cli/CliTest.java

```diff
@@ -0,0 +1,1167 @@
+/*
+ * Copyright (c) 2017-2018 Aion foundation.
+ *
+ *     This file is part of the aion network project.
+ *
+ *     The aion network project is free software: you can redistribute it
+ *     and/or modify it under the terms of the GNU General Public License
+ *     as published by the Free Software Foundation, either version 3 of
+ *     the License, or any later version.
+ *
+ *     The aion network project is distributed in the hope that it will
+ *     be useful, but WITHOUT ANY WARRANTY; without even the implied
+ *     warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *     See the GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with the aion network project source files.
+ *     If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ *     Aion foundation.
+ */
+package org.aion.zero.impl.cli;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.aion.util.TestResources.TEST_RESOURCE_DIR;
+import static org.aion.zero.impl.cli.Cli.ReturnType.ERROR;
+import static org.aion.zero.impl.cli.Cli.ReturnType.EXIT;
+import static org.aion.zero.impl.cli.Cli.ReturnType.RUN;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doCallRealMethod;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.spy;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.io.Files;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Path;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.ArrayList;
+import java.util.List;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.aion.base.util.Hex;
+import org.aion.crypto.ECKey;
+import org.aion.crypto.ECKeyFac;
+import org.aion.mcf.account.Keystore;
+import org.aion.mcf.config.Cfg;
+import org.aion.zero.impl.cli.Cli.ReturnType;
+import org.aion.zero.impl.config.CfgAion;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/** CliTest for new version with use of different networks. */
+@RunWith(JUnitParamsRunner.class)
+public class CliTest {
+
+    private final Cli cli = new Cli();
+    private final CfgAion cfg = CfgAion.inst();
+    private final Cli mockCli = spy(cli);
+
+    // base paths
+    private static final String BASE_PATH = System.getProperty("user.dir");
+    private static final File MAIN_BASE_PATH = new File(BASE_PATH, "mainnet");
```

Comment:
You could also use JUnit's TemporaryFolder to generate a temporary folder and set your base paths to that folder.  Then you won't need to manually delete stuff in your `@After `

Reply:
I created an issue for this refactoring #660